### ISSUE TITLE
Steno writes in Qwerty on my Azerty system

### ISSUE THREAD

**Issue by Rom1deTroyes** [keyboardio/Kaleidoscope#1091]
**Describe the bug**
I try to use the Steno plugin with my Keyboard.io Atreus, but it seems I have the same kind of problem as #1011 :
The words are written with Azerty <-> Qwerty inversion ( a->q z->w ...).

![image](https://user-images.githubusercontent.com/8625432/142883396-570631c1-8342-4405-a184-b66b652dc45b.png)

As a beginner in the Steno world, I may have missed something in the Plover config, so can someone confirm/infirm this ?

**To Reproduce**

- Make flash the Steno.ino example sketch
- Connect Plover with GeminiPR
- See the keyboard write strange words https://joshuagrams.github.io/steno-jig/learn-keyboard.html :
  - S(A) whites "q"
  - S(PL) writes "qbout"
  

**Expected behavior**
I expect Steno to write text as typed :-)

**Environment (please complete the following information):**
 - OS: GNU/Linux
 - Version : Debian10
 - Device Keyboardio Atreus

 - Firmware compiled from source (master) : the Atreus example, including the Steno stuff from the steno example, with keymap cutted to Atreus number of keys.
 - Plover 3.1.1 with GeminiPR Machine connected

**Additional context**

I try several searches about Plover and azerty, but find nothing relevant ATM...
Will try to have some support from the Plover community tomorrow, and keep in touch !

**Comment by Rom1deTroyes** [comment]
After a lot of little research and some pointers from Plover user to an  openstenoproject/plover#1395 I confirm that the problem is not between the keyboard and the OS :-)

tldr, you can test it with :
```bash
$ setxkbmap -query
 rules:      evdev
 model:      pc105
 layout:     fr
 variant:    latin9
$ xdotool type azerty
 qwerty
$ setxkbmap fr && xdotool type azerty
 azerty
```
